File: Course 4/Mobile Programming/phong/app/src/main/java/ru/mikheev/phong/MainActivity.java
package ru.mikheev.phong;

import androidx.appcompat.app.AppCompatActivity;

import android.opengl.GLSurfaceView;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    private MySurfaceView mGLSurfaceView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mGLSurfaceView = new MySurfaceView(this);
        setContentView(mGLSurfaceView);
    }
}

File: Course 3/Mobile Programming/Lab2/app/src/main/java/com/example/lab2/MainActivity.kt
package com.example.lab2

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        button.setOnClickListener { startActivity(Intent(this, picture::class.java)) }
        button2.setOnClickListener { startActivity(Intent(this, table::class.java)) }
        button3.setOnClickListener { startActivity(Intent(this, list::class.java)) }
    }
}
